cmd/veneur/main.go

```diff
@@ -2,11 +2,10 @@ package main
 
 import (
 	"flag"
-	"net"
 	"sync"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
```

Comment:
we've always been using logrus in veneur, if that's what you mean? I just removed the rename